storybookjs/storybook
# Addon-a11y: Lazy load axe-core only when running tests

Issue: #16456

## What I did
Lazy load axe-core so it is not fetched until a test is run.

## How to test
1. notice that the bundle size of built example storybooks are smaller (for example, angular-cli is >1MB smaller before gzipping)
2. turn on "manual" a11y parameter and watch for a network request when you run a test for the first time.

No
No
No
